### ISSUE TITLE
Last update broke insomnia

### ISSUE THREAD

**Issue by ppazos** [getinsomnia/insomnia#5579]
### Expected Behavior

Been using insomnia for days, now today it just closed itself, when I try to run it manually I see this error:

A JavaScript error occurred in the main process
Uncaught Exception:
Error: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.26' not found (required by /tmp/.org.chromium.Chromium.aNnee7)
    at process.func [as dlopen] (node:electron/js2c/asar_bundle:2:1822)
    at Module._extensions..node (node:internal/modules/cjs/loader:1226:18)
    at Object.func [as .node] (node:electron/js2c/asar_bundle:2:2049)
    at Module.load (node:internal/modules/cjs/loader:1011:32)
    at Module._load (node:internal/modules/cjs/loader:846:12)
    at f._load (node:electron/js2c/asar_bundle:2:13328)
    at Module.require (node:internal/modules/cjs/loader:1035:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/opt/Insomnia/resources/app.asar/node_modules/@getinsomnia/node-libcurl/dist/Easy.js:5:18)
    at Module._compile (node:internal/modules/cjs/loader:1141:14)

Not sure what is going on, I didn't change anything here, just updated it as I normally do.

### Actual Behavior

Can't run insomnia.

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Additional Information

I'm on linux mint 19.3

### Insomnia Version

not sure  I can't run the app

### What operating system are you using?

Other Linux

### Operating System Version

linux mint 19.3

### Installation method

deb [trusted=yes arch=amd64] https://download.konghq.com/insomnia-ubuntu/ default all

**Comment by ppazos** [comment]
I needed to downgrade and the only option is to use the AppImage. I can confirm:

1. AppImage 2022.6.0 works like charm
2. AppImage 2022.7.0 is broken, this is the same as the deb I have installed via the konghq repo

So for anyone having this error, use https://github.com/Kong/insomnia/releases/tag/core%402022.6.0

**Comment by otherpaco** [comment]
plus 1

OS: **Ubuntu 22.04**
snap: **insomnia v2022.7.0 rev184**

`strings /usr/lib/x86_64-linux-gnu/libstdc++.so.6 | grep GLIBCXX`

shows that **GLIBCXX_3.4.26** is installed

**Comment by filfreire** [comment]
Closing as Duplicate of https://github.com/Kong/insomnia/issues/5531, fix is ready in https://github.com/Kong/insomnia/pull/5555

**Comment by filfreire** [comment]
@ppazos this should be fixed from 2022.7.2 onwards for snapcraft builds

as for debian packages - it is likely the case that due to your Linux Mint version (19.3) being based of Ubuntu Bionic (18.04) - you could be missing a more updated version of `libstdc++6` package.

Not sure how it works for Linux Mint, you might be able to find an equivalent on DuckDuckgo/Google to:
```
sudo add-apt-repository ppa:ubuntu-toolchain-r/test
sudo apt update
sudo apt-get install --only-upgrade libstdc++6
```

If you can find a way to update your `libstdc++6` locally it should also help with the `GLIBCXX_3.4.26' not found` error when booting up.
